analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/Tests/VerifierTest.cs

```diff
@@ -121,6 +121,46 @@ public class VerifierTest
             DummyCS.AddSnippet("//Empty").WithProtobufPath("Proto.pb")
                 .Invoking(x => x.Build()).Should().Throw<ArgumentException>().WithMessage("DummyAnalyzerCS does not inherit from UtilityAnalyzerBase.");
 
+#if NET
```

Comment:
I will update these tests to use the [DummyAnalyzerWithLocationMapping](https://github.com/SonarSource/sonar-dotnet/compare/master...%C4%8Daba/noncompliant-location-razor#diff-19deb9d4f4e84eafc77801a552074c68020cd963605a0d1032e3df18576a6990R29) analyzer. This should enable us to have more precise assertions.

Reply:
The comment above is related to https://github.com/SonarSource/sonar-dotnet/pull/7755#issuecomment-1672824364